adapters/33across/33acrosstest/exemplary/bidresponse-defaults.json

```diff
@@ -5,10 +5,10 @@
       {
         "id": "test-imp-id",
         "video": {
-          "w": 728, 
+          "w": 728,
           "h": 90,
           "protocols": [2],
-          "placement": 1,
+          "plcmt": 1,
```

Comment:
If we are using `plcmt` over `placement`, should we modify anything in `adapters/33across/33across.go`? As opposed to `video.Placement`, your adapter seems to not be using `video.Plcmt` at all. As opossed to what your adapter was doing with  `video.Placement`, we simply seem to pass `video.Plcmt` along. Is this intended?